README.md

```diff
@@ -1,323 +1,299 @@
 
-# Simulating clients
+To run the `Suite`, you can call either `RunSuite()`or `MustRunSuite()` depending on how you want errors to be handled. 
 
+* `RunSuite()` will run all tests in the `Suite`, returning an error upon failure. 
 
-----
-`hive` supports a more advanced form of client testing called *simulations*, where entire networks
-of clients are run concurrently under various circumstances and their behavior monitored and checked.
+* `MustRunSuite()` runs the given suite, exiting the process if there is a problem executing the test.
 
-Running network simulations is completely analogous to validations from the user's perspective: you
-can specify which clients to simulate with the `--client` flag, and you can specify which
-simulations to run via the `--sim` regexp flag. By default simulations aren't being run as they can
-be quite lengthy.
+Both Run functions take `Suite` as a parameter as well as a pointer to an instance of `hivesim.Simulation`, as such: 
 
+```go
+hivesim.MustRunSuite(hivesim.New(), suite)
+```
 
-`--client` is now an explicit list of permitted clients, where the desired client is of the format
-clientname_branch. For example `go-ethereum_latest` and `parity_beta`. Previously this was a regex. 
-Other parameters such as the sim or validator still use regexp. This change was introduced to allow
-for branches to be specified as parameters without the need for creating a dedicated client subfolder 
-for that branch. `go-ethereum` and `parity` in this example need to be subfolders of the Hive `clients`
-folder with their own dockerfile. The branch is parsed out and passed as a docker build arg.
+`hivesim.Simulation` is just a wrapper API that can access the hive server. To get a new instance of `hivesim.Simulation`, call `hivesim.New()`. This will look up the hive host server URI and connect to it
 
 
-Simulators now offer a golang client framework, that allows them to call into the Hive Simulator 
-API and create different types of client. The simulator can run tests or other experiments written in 
-Golang against one or more instances of clients. To achieve this, a number of new options are added:
+###### Getting information about the client container
 
-`--sim.rootcontext` a boolean, which when set tells the compiler to build the docker image with 'simulators'
-as the root of the context, allowing the simulators\common and simulators\devp2p common code to be included
-in the simulator. 
+To get information about the client that is likely necessary for test execution, you can use `hivesim.Client`within the aforementioned test execution function (`myTestFunction`).
 
-Sim.rootcontext needs to be set differently depending on the type of simulation being run. For the consensus tests
-the base simulator image relies on files to be added from a folder local to the image. For developing new simulations,
-or extending the existing ones, it is recommended to use sim-rootcontext as true. 
+**Enode ID**
```

Comment:
Enode ID is a bad term, and we should avoid using it. The correct term for this is 'enode URL'.